<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputParamsDialog</class>
 <widget class="QDialog" name="inputParamsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <layout class="QGridLayout" name="inputParamButtonLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="addParamButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="removeParamButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="inputParametersTable">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="inputParamtersLabel">
       <property name="text">
        <string>Input Parameters:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
